pkg/containerd/cmd/service/start.go

```diff
@@ -40,55 +41,65 @@ func start(args []string) {
 	}
 
 	service := args[0]
-	rootfs := filepath.Join("/containers/services", service, "rootfs")
+
 	log.Infof("Starting service: %q", service)
 	log := log.WithFields(log.Fields{
 		"service": service,
 	})
 
-	client, err := containerd.New(*sock)
+	id, pid, msg, err := start(service, *sock, *path, *dumpSpec)
+	if err != nil {
+		log.WithError(err).Fatal(msg)
+	}
+
+	log.Debugf("Started %s pid %d", id, pid)
+}
+
+func start(service, sock, path, dumpSpec string) (string, uint32, string, error) {
+	rootfs := filepath.Join(path, service, "rootfs")
+
+	client, err := containerd.New(sock)
 	if err != nil {
-		log.WithError(err).Fatal("creating containerd client")
+		return "", 0, "creating containerd client", err
```

Comment:
well, the string is part of error because of how `logrus` is returning errors as it wants `err` and the message seperately, I might just get rid of `logrus` to clean it up...